test/blackbox-tests/test-cases/patch-back-source-tree.t

```diff
@@ -175,12 +175,15 @@ If a source file is read-only, the action sees it as writable:
 
   $ dune build
   writable
+  Error: open(x): Permission denied
+  -> required by alias default in dune:1
+  [1]
```

Comment:
Is this an expected change? I would not expect anything in the expect tests to change just by using a more memory compact representation.

Reply:
Indeed, this change is wrong and not needed. Will remove.